cmd/extractDiffs.go

```diff
@@ -40,8 +42,23 @@ func init() {
 }
 
 func extractDiffs() {
+	operation := func() error {
+		err := extractDiffsOnce()
+		return err
+	}
+
+	b := backoff.NewExponentialBackOff()
+	b.MaxElapsedTime = 120 * time.Second
```

Comment:
Should we consider making this continuously retry?

Reply:
No, we should cap the max pause time. I think this is a good place to start.